display_information/identify-graphics/src/main/java/com/esri/samples/identify_graphics/IdentifyGraphicsSample.java

```diff
@@ -80,7 +80,7 @@ public void start(Stage stage) {
       mapView.getGraphicsOverlays().add(graphicsOverlay);
 
       // work with the MapView after it has loaded
-      mapView.addSpatialReferenceChangedListener(src -> addGraphicsOverlay());
+      mapView.spatialReferenceProperty().addListener(e -> addGraphicsOverlay());
```

Comment:
I'm not convinced why this is actually being used. The `addGraphicsOverlay` method also says it's adding 4 graphics but it's adding one. I think we could just create the graphics and add them to the graphics overlay before adding the graphics overlay to the mapview. I'll look at this sample in a bit more detail.